components/form/pta/src/index.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+export const BASE_CLASS = 'sui-FormPta'
```

Comment:
You're not using this const. You may want to use it to build the classnames in the `render`.

Reply:
I don't need to export it ;-)